tests/garage/tf/models/test_mlp_model.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.models import MLPModel
+from tests.fixtures import TfGraphTestCase
+
+
+class TestMLPModel(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.input_var = tf.placeholder(tf.float32, shape=(None, 5))
+        self.obs = np.ones((1, 5))
+        self.output_dim = 1
+
+    def test_output_values(self):
+        model = MLPModel(
+            output_dim=self.output_dim,
+            hidden_sizes=(2, ),
```

Comment:
what about hidden_size = 1?

in most software, bugs center around the values 0, 1, 2, and 3+. it's good to test all of those cases (or whichever ones are valid for your test)